### DISCUSSION TITLE
Recreate the Pride School Confetti

### DISCUSSION THREAD

**Discussion by JaavierR** [tsparticles/vue3#32] [Q&A]
Hi, I am trying to recreate the effect of [Pride School Confetti](https://github.com/tsparticles/confetti/blob/7dd81f820b1a14af8d96c93febef3a931ffaed4b/public/js/more.js#L510) but I have the doubt if it is possible or not with this component, as it seems that the example of the site is based on a specific bundle to generate confetti. 

If I load the cdn link and run the example code it works without problems, but I have not been able to recreate it with vue, and I have not found any example that could guide me. Has anyone been able to recreate this effect that can give me some help or refer me somewhere in the documentation that I have overlooked?

**Comment by JaavierR** [comment]
I just realised that I totally overlooked the [tsparticles-confetti library](https://www.npmjs.com/package/tsparticles-confetti). With it I was able to achieve what I needed and I doubt that you need to use the vue component thanks to that library, so I'll close the subject thanks anyway and I hope it will be useful to someone in the future.

**Reply by matteobruni** [reply]
I created a component in angular for that library [here](https://github.com/tsparticles/angular/tree/main/components/confetti).

Maybe it's something that can be brought to other frameworks too.
